kraken-wrapper/pyproject.toml

```diff
@@ -20,7 +20,7 @@ Homepage = "https://kraken-build.github.io/kraken-build/"
 Repository = "https://github.com/kraken-build/kraken-build/"
 
 [tool.poetry.dependencies]
-python = ">=3.7"
+python = ">=3.7,<3.11"
```

Comment:
```suggestion
python = ">=3.7"
```

Reply:
poetry doesn't like this. This is because:
1. there are dependencies that don't support python 4.
2. there are dependencies that don't support python 3.12
3. I believe that kraken breaks if you're on 3.11 anyway? although the wrapper might be fine.

Reply:
Python 4??

The wrapper is indeed fine on 3.11 and just yesterday I made the PR because someone was facing the issue about not being able to install it into 3.11. The wrapper has no issue with 3.11 so I considered the upper limit a bug.

If there are dependencies that don't support 3.12 we can still add back `<3.12` until they support it.

Reply:
ok i'll make it <3.12. It is indeed an issue to just do ^3.7, because poetry picks dependencies that are valid for all versions in the range.